### DISCUSSION TITLE
Python script works well in local system，but SIGTERM in singularity images.

### DISCUSSION THREAD

**Discussion by jiangzhangze** [sylabs/singularity#1680] [Q&A]
**Type of issue**
test problem

**Description of issue**
Hi ，@preminger.I try to train a [subprocess](https://stable-baselines3.readthedocs.io/en/master/guide/vec_envs.html#subprocvecenv) environment reinforcement learning task by stable-baselines3.Script works in local system, but trainning is usually interrupted by SIGTRM when put the python script into singularity images and no error output.
What should I do to make my singularity images can work as same as my local systerm.

**Comment by dtrudg** [comment]
This is a usage question rather than a bug report, so I'll transfer it to GitHub discussions.

As noted in Slack, you need to provide more information about how you are running the Singularity images. Is this on a cluster, through a scheduler like Slurm etc.?

**Reply by jiangzhangze** [reply]
No，it is ran on a local machine of Ubuntu20.04 .The version of singularity is 3.11.
In addtion，this problem only happened when I runs each environment in a separate process，and works well in a single process.And it works well in my local machine:Both in separate process and single process.

**Reply by dtrudg** [reply]
Are there resource limits / process limits on the machine where it fails? Can you give us a bit more information about that environment? What exactly is the machine where it fails? How does the configuration of that machine differ from your local machine? Without that information this will be very difficult to answer.
